CONTRIBUTING.md

```diff
@@ -19,12 +19,22 @@ swift-syntax is a SwiftPM package, so you can build and test it using anything t
 
 swift-syntax is formatted using [swift-format](http://github.com/apple/swift-format) to ensure a consistent style.
 
-To format your changes run `format.py` at the root of this repository. If you have a `swift-format` executable ready, you can pass it to `format.py`. If you do not, `format.py` will build its own copy of `swift-format` in `/tmp/swift-format`.
-
-If you are seeing surprising formatting results, you most likely have a `swift-format` installed on your system that’s not the most recent version built from the `main` branch. To fix this, clone [swift-format](http://github.com/apple/swift-format), build it using `swift build` and pass the freshly built executable to `format.py` as `--swift-format path/to/swift-format/.build/debug/swift-format`. Alternatively, you can uninstall `swift-format` on your system and `format.py` will build it from scratch.
+To format your changes run `format.swift` at the root of this repository from the terminal as `./format.swift`. It will build a local copy of swift-format from the `main` branch and format the repository. Since it is building a release version of `swift-format`, the first run will take few minutes. Subsequent runs take less than 2 seconds.
```

Comment:
```suggestion
To format your changes, run `format.swift` at the root of this repository from the terminal as eg. `./format.swift`. It will build a local copy of swift-format from the `main` branch and format all files. Since it is building a release version of `swift-format`, the first run will take few minutes. Subsequent runs are much faster (a couple seconds).
```